department-of-veterans-affairs/caseflow
# Correspondence Cases - Unassigned Tab - Superuser | Remove Select Column

Resolves [Correspondence Cases - Unassigned Tab - Superuser | Remove Select Column](https://jira.devops.va.gov/browse/APPEALS-42300)

# Description
Previously the queuetable for SuperUser was populating with the checkbox column that should only be available to supervisors. This code change fixes this by conditionally adding the checkbox column based on mail_supervisor? and mail_superuser?.

## Acceptance Criteria

## Testing Plan

1. Login as AMBRISVACO
2. Navigate to the unassigned tab in the org cases
3. Note that the checkbox is no longer appearing for the user
4. login as MAIL_TEAM_SUPERVISOR_ADMIN_USER (VACO) (who is a supervisor)
5. navigate to the unassigned tab in the org cases
6. Note that the checkbox still appears.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
<img width="1245" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/12f600af-5d2e-4836-ab4f-b8c072f0109c">
|
<img width="1259" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/deaab8ff-c59e-49db-8a14-4aa32a83581f">
